R/convergence.R

```diff
@@ -316,6 +316,144 @@ quantile2 <- function(x, probs = c(0.05, 0.95), names = TRUE, ...) {
   out
 }
 
+#' Calculate R star convergence diagnostic
+#'
+#' The `rstar` function generates a measure of convergence for MCMC draws based on
+#' whether it is possible to determine the Markov chain that generated a draw with
+#' probability greater than chance. To do so, it fits a machine learning classifier
+#' to a training set of MCMC draws and evalutes its predictive accuracy on a testing
+#' set: giving the ratio of accuracy to predicting a chain uniformly at random.
+#'
+#' @param x a `draws_df` object or one coercible to a `draws_df` object.
+#'
+#' @param split_chains a Boolean indicating whether to split chains into two equal
+#' halves.
+#'
+#' @param uncertainty whether to provide a vector of R* values (if true) or a single
+#' value (if false).
+#'
+#' @param method machine learning classifer available in caret R package.
+#'
+#' @param hyperparameters hyperparameter settings for classifier given as a list.
+#'
+#' @param nsim number of R* values in returned vector if uncertainty is true.
```

Comment:
I think it would be more in line with posterior naming conventitions to call the argument `nsimulations`

Reply:
Done.